apps/onboarding/src/wallet/wallet.service.ts

```diff
@@ -131,14 +154,21 @@ export class WalletService {
     }
 
     const impl = newMetaTransactionWallet(this.web3, implementationAddress)
+    const txn = toRawTransaction(
+      this.walletDeployer.deploy(
+        session.externalAccount,
+        implementationAddress,
+        impl.methods.initialize(session.externalAccount).encodeABI()
+      ).txo
+    )
     const resp = await this.relayerProxyService.submitTransaction({
-      transaction: toRawTransaction(
-        this.walletDeployer.deploy(
-          session.externalAccount,
-          implementationAddress,
-          impl.methods.initialize(session.externalAccount).encodeABI()
-        ).txo
-      )
+      transaction: txn
+    })
+
+    this.logger.logWalletDeployed({
```

Comment:
Let's actually call this event `DeployWalletTxSent` because we just have a tx hash at this point and can't know if that deploy will end up happening or reverting.